spec/integration/admin_api/apis_routes_spec.lua

```diff
@@ -109,6 +110,7 @@ describe("Admin API", function()
         assert.truthy(body_page_1.data)
         assert.equal(3, table.getn(body_page_1.data))
         assert.truthy(body_page_1.next)
+        assert.equal(10, body_page_1.total)
```

Comment:
The function is called `"should retrieve a paginated set"`, so I am checking the response is a valid paginated set, and that also includes the `total` field.

If the tests themselves need to be changed, well that's another thing, in this specific case I integrated the new feature check with the existing tests.